### PR TITLE
Add heading in contents for screen reader

### PR DESCRIPTION
**Pull request by shawn320**
# Description
This pr fixes the issue #231

Added headings for screen reader. Added ```<h2>``` heading in Header and Footer and ```<h3>``` headings in the navigations. The heading will not display in the browser but will be announced by the screen reader.

Heading titles and French translation:
Screen Shot 2021-08-16 at 2 53 17 PM

### CODE REVIEW

File: components/organisms/Layout.js
Comment (will0684) [inline components/organisms/Layout.js:46]: Nice work, I think these strings should be bilingual

**Review by will0684 — CHANGES_REQUESTED** [review]

File: components/organisms/Layout.js
Comment (will0684) [reply components/organisms/Layout.js:46]: Message Isabelle Waltzing with what you would like translated, she is probably best to provide them.

Comment (shawn320) [reply components/organisms/Layout.js]: Thank you, I'll message her for the translate.

**Review by will0684 — CHANGES_REQUESTED** [review]
Waiting on review from Isabelle

**Review by will0684 — APPROVED** [review]
Good stuff thanks Shawn 👍

**Pull request merged**
